composer.json

```diff
@@ -32,6 +32,7 @@
         }
     ],
     "require": {
+        "php": ">=7.0",
         "neos/neos": "~3.0.4 || ^3.1.2",
```

Comment:
remove the php requirement and change the `neos/neos` requirement to `^3.2`

Reply:
Done :)